src/tiled/propertybrowser.h

```diff
@@ -151,8 +154,11 @@ private slots:
                                       const QString &name,
                                       QtProperty *parent);
 
+    void addNewProperties(Object* object);
+    void removeCurrentProperties();
```

Comment:
Naming-wise, I'd call these functions simply `addProperties` and `removeProperties`. The "new" and "current" additions are not adding any information. Also, the `addNewProperties` function does not need to take the `Object*` parameter, but should just use `mObject`.